features/freaks/src/main/kotlin/com/agilefreaks/freaks_catalog/features/freaks/FreakD.kt

```diff
@@ -0,0 +1,14 @@
+package com.agilefreaks.freaks_catalog.features.freaks
+
+data class FreakD(
```

Comment:
This abbreviation doesn't have sense here. We can keep it as FreakDetails. Clean and simple